pkgs/development/libraries/mesa/default.nix

```diff
@@ -125,6 +128,11 @@ let
   haveDozen = (lib.elem "d3d12" galliumDrivers) || (lib.elem "microsoft-experimental" vulkanDrivers);
 
   rustDeps = [
+    {
+      pname = "paste";
+      version = "1.0.14";
+      hash = "sha256-+J1h7New5MEclUBvwDQtTYJCHKKqAEOeQkuKy+g0vEc=";
+    }
```

Comment:
Do any of the other crates need updated?

Reply:
No, meson would scream if they did.